src/collections/_documentation/enriching-error-data/set-extra/go.md

```diff
@@ -1,5 +1,11 @@
 ```go
 sentry.ConfigureScope(func(scope *sentry.Scope) {
 	scope.SetExtra("{{ page.example_extra_key }}", "{{ page.example_extra_value }}")
+
+    // to set multiple key-value pairs at once:
+    scope.SetExtras(map[string]interface{}{
```

Comment:
Indentation (go uses tabs, not spaces, thus this difference from what i see)

Reply:
✔️